src/create-cluster.ts

```diff
@@ -48,10 +46,10 @@ export async function createCluster(
     startArgs.push('--mount-string', mountString);
   }
 
-  // update PATH to include minikube
-  env.PATH = getMinikubePath();
-
-  env.MINIKUBE_HOME = getMinikubeHome();
+  const env: Record<string, string> = {
+    PATH: getMinikubePath(),
+    MINIKUBE_HOME: getMinikubeHome() ?? '',
```

Comment:
should it be

if it exists we add it to env object else we don't rather than adding an empty string value

Reply:
Yes, I agree, fixed 👍